lib/compile.ml

```diff
@@ -5,15 +5,12 @@ let compile_fragment all_infos info =
   (* Get the filename of the output odoc file *)
   let odoc_path = Inputs.compile_target info in
 
-  let odoc_path_result = Fpath.segs odoc_path |> List.filter (fun x -> x <> "odoc-pages") |> String.concat "/" |> Fpath.of_string in
-  let odoc_path = match odoc_path_result with | Ok r -> r | _ -> failwith "error" in
```

Comment:
This seems not to be needed at all because it doesn't change paths created by `odoc html`.
So instead of moving it to `Inputs`, I removed this detail.